framework/src/modules/chain/transaction_pool/transaction_pool.js

```diff
@@ -179,7 +156,9 @@ class TransactionPool extends EventEmitter {
 						`${queueName} size: ${this.pool._queues[queueName].size()}`,
 				)
 				.join(' ');
-			this.logger.info(`Transaction pool - ${queueSizes}`);
+			if (queueSizes.length > 0) {
```

Comment:
queue size is always greater than 0 🤔

Reply:
most of the time the queue size 0, so it keeps logging this info message, so I am checking if its greater and then only print.

Reply:
queueSize.length is the string, and it's based on the number of kinds of the queues